x-pack/plugins/observability_solution/observability_shared/common/field_names/elasticsearch.ts

```diff
@@ -145,3 +145,12 @@ export const PROFILE_ALLOC_OBJECTS = 'profile.alloc_objects.count';
 export const PROFILE_ALLOC_SPACE = 'profile.alloc_space.bytes';
 export const PROFILE_INUSE_OBJECTS = 'profile.inuse_objects.count';
 export const PROFILE_INUSE_SPACE = 'profile.inuse_space.bytes';
+
+export const ENTITY = 'entity';
+export const ENTITY_ID = 'entity.id';
+export const ENTITY_TYPE = 'entity.type';
+export const ENTITY_LAST_SEEN = 'entity.lastSeenTimestamp';
+export const ENTITY_FIRST_SEEN = 'entity.firstSeenTimestamp';
```

Comment:
`firstSeenTimestamp` will be removed once https://github.com/elastic/kibana/pull/193999 is merged. we have plans to reintroduce it but might have to wait until next release. Is this field required for the inventory view ?

Reply:
No, it's there just because of APM. We'll also remove APM code soon. So no worries about this.